bolero/representation/test/test_dmp_seq.py

```diff
@@ -163,9 +163,18 @@ def test_trajectory_generation():
     assert_almost_equal(traj[100,0], subgoal, places=2)
 
 
+def test_subgoal_velocities():
+    test_task_dims = range(1,100,1)
+    for n_task_dims in test_task_dims:
+        dmp_seq, subgoals = create_dmp_seq(n_task_dims=n_task_dims)
+        dmp_seq.set_params(dmp_seq.get_params())
+        for i in range(len(subgoals)):
+            assert_equal(len(dmp_seq.get_subgoal_velocity(i)),n_task_dims)
```

Comment:
Please add a space after ","

You can automatically check PEP8 compliance of a file with the command line tools pep8 or flake8 (can be installed with pip).

Reply:
Thanks, installed the tool pycodestyle as the use of the pep8 tool will be removed in a future release:

1.7.1 (2017-10-22)
Changes:
- Prominently note via warning message that the tool is no longer released as pep8 and will only be fixed in the pycodestyle package

Reply:
see [d45cdbc](https://github.com/rock-learning/bolero/pull/44/commits/d45cdbc499cd458ac97f506f4a8303d02b9d452b)